app/views/organisers/index.html.slim

```diff
@@ -5,6 +5,13 @@
       p
       p
       p
+      = form_tag organisers_path, :method => :post do
```

Comment:
How to display error messages using form_tag? I am getting a way to do it using for_for which have user object attached to it. But I have no clue How to send validation error messages to view using form_tag? Also How you display error messages ? Using :message in validates, or making another fn ?